observe/observe.go

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package observe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	proto "github.com/kubearmor/koach/protobuf"
+	"github.com/rodaine/table"
+	"google.golang.org/grpc"
+)
+
+// Options Structure
+type Options struct {
+	Operation     string
+	Namespace     string
+	AllNamespace  bool
```

Comment:
If above option(Namespace) is null/empty, can we cover all namespaces ?

Reply:
no.

the behaviour is the same kubectl, if it is empty, then we will assume it is "default" namespace, I have provided "-A" flag to cover all namespaces